packages/wallets/src/features/cashier/modules/WithdrawalCrypto/components/WithdrawalCryptoForm/WithdrawalCryptoForm.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect } from 'react';
+import { Field, FieldProps, Formik } from 'formik';
+import { useActiveWalletAccount, useCryptoWithdrawal, useCurrencyConfig, useExchangeRate } from '@deriv/api';
+import { WalletButton, WalletTextField, WalletsPercentageSelector } from '../../../../../../components';
+import { WithdrawalCryptoAmountConverter } from './components/WithdrawalCryptoAmountConverter';
+import './WithdrawalCryptoForm.scss';
+
+const MIN_ADDRESS_LENGTH = 25;
+const MAX_ADDRESS_LENGTH = 64;
+
+export type TForm = {
+    cryptoAddress: string;
+    cryptoAmount: string;
+    fiatAmount: string;
+};
+
+type TWithdrawalCryptoFormProps = {
+    verificationCode?: string;
+};
+
+const validateCryptoAddress = (address: string) => {
+    if (!address) return 'This field is required.';
+
+    if (address.length < MIN_ADDRESS_LENGTH || address.length > MAX_ADDRESS_LENGTH) {
+        return 'Your wallet address should have 25 to 64 characters.';
+    }
+
+    return undefined;
+};
+
+const WithdrawalCryptoForm: React.FC<TWithdrawalCryptoFormProps> = ({ verificationCode }) => {
+    const { data: activeWallet } = useActiveWalletAccount();
+    const { getConfig } = useCurrencyConfig();
+    const { data: exchangeRate, subscribe, unsubscribe } = useExchangeRate();
+    const { mutate } = useCryptoWithdrawal();
+    const FRACTIONAL_DIGITS_CRYPTO = activeWallet?.currency ? getConfig(activeWallet?.currency)?.fractional_digits : 2;
+    const FRACTIONAL_DIGITS_FIAT = getConfig('USD')?.fractional_digits;
+
+    useEffect(() => {
+        if (activeWallet?.currency)
+            subscribe({
+                base_currency: 'USD',
+                loginid: activeWallet.loginid,
+                target_currency: activeWallet.currency,
+            });
+        return () => unsubscribe();
+    }, []);
```

Comment:
Missing dependencies on the `useEffect`

Reply:
This is done to subscribe to the `exchange_rate` call only when the component mounts. So we don't need any dependencies as the hooks needs to be just fired once.